lib/v2/tingshuitz/changsha.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const baseUrl = 'http://www.supplywater.com';
+
+module.exports = async (ctx) => {
+    const { channelId = 78 } = ctx.params;
+    const listPage = await got('http://www.supplywater.com/tstz-' + channelId + '.aspx');
+    const $ = cheerio.load(listPage.data);
+    const pageName = $('.mainRightBox .news-title').text();
+    const list = $('.mainRightBox .announcements-title a')
+        .map((_, item) => {
+            item = $(item);
+
+            return {
+                title: item.text().trim(),
+                link: baseUrl + item.attr('href').trim(),
+            };
+        })
+        .get();
+
+    const items = await Promise.all(
+        list.map(async (item) => {
+            const postPage = await got(item.link);
```

Comment:
Any reasons why cache is not used?

Reply:
Well, it's notice of water supply being cut off. The posts are often updated about the progress, like the new estimated recovery time or recovery announcement. So I think it's better to leave the RSS client to decide the fetch interval.